libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -495,6 +495,70 @@ void RC_Channel::init_aux_function(const aux_func_t ch_option, const AuxSwitchPo
     }
 }
 
+#if !HAL_MINIMIZE_FEATURES
+
+const RC_Channel::LookupTable RC_Channel::lookuptable[] = {
+    { AUX_FUNC::SIMPLE_MODE,"SimpleMode"},
+    { AUX_FUNC::SAVE_TRIM,"SaveTrim"},
+    { AUX_FUNC::SAVE_WP,"SaveWaypoint"},
+    { AUX_FUNC::CAMERA_TRIGGER,"CameraTrigger"},
+    { AUX_FUNC::RANGEFINDER,"Rangefinder"},
+    { AUX_FUNC::FENCE,"Fence"},
+    { AUX_FUNC::RESETTOARMEDYAW,"ResetToArmedYaw"},
+    { AUX_FUNC::SUPERSIMPLE_MODE,"SuperSimpleMode"},
+    { AUX_FUNC::SPRAYER,"Sprayer"},
+    { AUX_FUNC::GRIPPER,"Gripper"},
+    { AUX_FUNC::PARACHUTE_ENABLE,"ParachuteEnable"},
+    { AUX_FUNC::PARACHUTE_RELEASE,"ParachuteRelease"},
+    { AUX_FUNC::PARACHUTE_3POS,"Parachute3Position"},
+    { AUX_FUNC::MISSION_RESET,"MissionReset"},
+    { AUX_FUNC::RETRACT_MOUNT,"RetractMount"},
+    { AUX_FUNC::RELAY,"Relay1"},
+    { AUX_FUNC::LANDING_GEAR,"Landing"},
+    { AUX_FUNC::MOTOR_INTERLOCK,"MotorInterlock"},
+    { AUX_FUNC::RELAY2,"Relay2"},
+    { AUX_FUNC::RELAY3,"Relay3"},
+    { AUX_FUNC::RELAY4,"Relay4"},
+    { AUX_FUNC::AVOID_ADSB,"AvoidADSB"},
+    { AUX_FUNC::PRECISION_LOITER,"PrecisionLoiter"},
+    { AUX_FUNC::AVOID_PROXIMITY,"AvoidPRoximity"},
+    { AUX_FUNC::WINCH_ENABLE,"WinchEnable"},
+    { AUX_FUNC::WINCH_CONTROL,"WinchControl"},
+    { AUX_FUNC::RC_OVERRIDE_ENABLE,"RCOverrideEnable"},
+    { AUX_FUNC::USER_FUNC1,"User1"},
+    { AUX_FUNC::USER_FUNC2,"User2"},
+    { AUX_FUNC::USER_FUNC3,"USer3"},
```

Comment:
The User-hooks don't actually do anything

Reply:
I can agree that we could drop these for now until they are used...btw, dont know if the pre-arm check will catch them as being "do nothings"